### PR TITLE
Updated Troubleshooting

### PR DESCRIPTION
**Pull request by Chandler-Kluser**
As requested in https://github.com/trabucayre/openFPGALoader/issues/250#issuecomment-1238577926 I have updated troubleshooting file with some issues solved.

### CODE REVIEW

**Comment by trabucayre** [comment]
First all thanks for your contribution.
See below for some remarks

File: doc/guide/troubleshooting.rst [hunk 1]
```diff
@@ -14,3 +14,61 @@ Gowin device could not communicate since last bitstream flashed. (issue `#206 <h
...
 Gowin's FPGA may fails to be detected if **JTAGSEL_N** (pin 08 for *GW1N-4K*) is used as a GPIO.
 To recover you have to pull down this pin (before power up) to recover JTAG interface (*UG292 - JTAGSELL_N section*).
+
+JTAG init failed
+================
+
+Avoid using USB hubs like these:
+
+.. image:: media/usb_hub.jpg
+    :align: center
```
Comment (trabucayre) [inline doc/guide/troubleshooting.rst:24]: Picture is necessary? What about copyright?
USB hubs may, or not, an issue. I use more or less all the time thinkpad dock's hub without any issue (but it's not a USB powered hub).
Maybe it's better to explain to try without hub to see if it fix issue?

File: doc/guide/troubleshooting.rst [hunk 2]
```diff
@@ -14,3 +14,61 @@ Gowin device could not communicate since last bitstream flashed. (issue `#206 <h
...
+
+    Bus 002 Device 004: ID 0403:6010 Future Technology Devices International, Ltd FT2232C/D/H Dual UART/FIFO IC
+                           \__/ \__/ \________________________________________________________________________/
+                            |     |                     |
+                            |     |                     \-----> Device Description Name
+                            |     \---------------------------> idProduct
+                            \---------------------------------> idVendor
+
+
+in this case, you will have to add/edit your udev rule file to:
```
Comment (trabucayre) [inline doc/guide/troubleshooting.rst:70]: An udev rules file is provided at root directory of this repository and there is a basic
[explain](https://trabucayre.github.io/openFPGALoader/guide/install.html#udev-rules) description. Maybe a link to this section is required?
I think group stuff may be improved (to explain it's required to logout/login for update taken into account or using `sudo - $USER` for force the new group (with `id` to check).

**Comment by Chandler-Kluser** [comment]
@trabucayre modifications done

**Comment by trabucayre** [comment]
Applied to master. Thanks!

**Pull request merged**